ci/helm-chart/README.md

```diff
@@ -65,6 +65,7 @@ and their default values.
 | extraArgs | list | `[]` |  |
 | extraConfigmapMounts | list | `[]` |  |
 | extraContainers | string | `""` |  |
```

Comment:
Again should this be type list? Looking at the whole file, you would need to define a list of objects which contain extra container properties.

Reply:
I think this is better to be a string, which is used as a template rather than a list of objects. It coudl theoretically be made like for example `extraConfigmapMounts` - as a list of objects. However I would not recommend it, firstly because container is a much more complicated structure than a confgmap mounts. Mounts only have couple of fields, which are unlikely to be much changed in the future. Container have a lot more stuff in there and trying to strictly cover all of stuff possible in there is too complicated (I will not be doing such a PR myself for sure) and would also have to be changed each time k8s introduces new stuff possible in the containers. Keeping this as a template (string, containing templated yaml) is much more flexible.

Reply:
I agree, you're right. Can you add an example then we should be good to go?